test-cases/longevity/longevity-lwt-24h-multidc.yaml

```diff
@@ -0,0 +1,29 @@
+test_duration: 1560
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data_multidc.yaml n=10000000 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=1000" ]
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data_multidc.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1,lwt_deletes=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=30" ]
+stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data_multidc.yaml ops'(select=1)' cl=SERIAL duration=1440m -mode native cql3 -rate threads=30" ]
+
+n_db_nodes: '4 3 2'
+n_loaders: 3
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.xlarge'
+
+# loader AMI with c-s ver. 4 and few fixes:
+#  - fix NoSuchElementException
+#  - enable control over both consistency levels: regular and serial
+#  - bring shard awareness
+regions_data:
```

Comment:
just a small nitpick:
how many test are using this ? maybe we should move the c-s 4 version loader into it's own yaml ?

Reply:
@juliayakovlev please open a trello item for put this in one place. o.k. ?

Reply:
@fruch  Sure